.grabl/automation.yml

```diff
@@ -73,3 +73,10 @@ build:
         sudo systemd-run ./dist/typedb-all-linux/typedb server
         sleep 60
         bazel test //xcom:test --test_output=errors
+    build-gitapp:
+      image: vaticle-ubuntu-21.04
+      command: |
+        bazel run //:typedb-extractor -- dist/typedb-all-linux
+        sudo systemd-run ./dist/typedb-all-linux/typedb server
+        sleep 60
+        bazel build //github:github-bin-linux
```

Comment:
This should be a test target. If we don't have tests already, we should add some!